### PR TITLE
removed depreciated terminology from certificate challenges

### PR DESCRIPTION
**Pull request by AkiraLaine**

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/front-end-development-certificate.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         [
           "http://i.imgur.com/syJxavV.jpg",
           "An image of our Front End Development Certificate",
-          "This challenge will give you your verified Front End Development Certificate. Before we issue your certificate, we must verify that you have completed all of our basic and intermediate Bonfires, and all our basic and intermediate Ziplines. You must also accept our Academic Honesty Pledge. Click the button below to start this process.",
+          "This challenge will give you your verified Front End Development Certificate. Before we issue your certificate, we must verify that you have completed all of our basic and intermediate algorithm scripting chballenges, and all our basic and intermediate front end development projects. You must also accept our Academic Honesty Pledge. Click the button below to start this process.",
```
Comment (QuincyLarson) [inline seed/challenges/01-front-end-development-certification/front-end-development-certificate.json]: You misspelled "challenges"

File: seed/challenges/03-back-end-development-certification/back-end-development-certificate.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
         [
           "http://i.imgur.com/2qn7tHp.jpg",
           "An image of the text \"Back End Development Certificate requirements\"",
-          "Let's confirm that you have completed our upper intermediate and advanced Bonfires, and our Basejumps. Click the button below to verify this.",
+          "Let's confirm that you have completed our upper intermediate algorithm scripting challenges, and our back end development projects. Click the button below to verify this.",
```
Comment (QuincyLarson) [inline seed/challenges/03-back-end-development-certification/back-end-development-certificate.json]: They no longer need to complete the upper intermediate algorithm scripting challenges - please drop this entirely and just say "back end development projects"

File: seed/challenges/03-back-end-development-certification/back-end-development-certificate.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
         [
           "http://i.imgur.com/sKYQhdG.jpg",
           "An image of our Back End Development Certificate",
-          "This challenge will give you your verified Back End Development Certificate. Before we issue your certificate, we must verify that you have completed all of Bonfires, Ziplines and Basejumps. You must also accept our Academic Honesty Pledge. Click the button below to start this process.",
+          "This challenge will give you your verified Back End Development Certificate. Before we issue your certificate, we must verify that you have completed all of the algorithm scripting challenges, front end development projects and back end developent projects. You must also accept our Academic Honesty Pledge. Click the button below to start this process.",
           ""
```
Comment (QuincyLarson) [inline seed/challenges/03-back-end-development-certification/back-end-development-certificate.json]: I didn't catch this earlier. It should just say "completed all of the back end development projects"

**Comment by SaintPeter** [comment]
:+1:

**Pull request merged**